### ISSUE TITLE
deal with change in page image resolution change

### ISSUE THREAD

**Issue by jeremybmerrill** [tabulapdf/tabula#253]
Or, regenerate the thumbnails at a higher resolution. Or something.

**Comment by jazzido** [comment]
No nuking, please. Additionaly, I don't think that regenerating page thumbnails is worth the implementation effort nor the processing time (BTW, once we get 1.0 out the door, let's move to PDF.js. Generating thumbnails sucks).

If anything, let's start fresh. That is, use a different directory name but keep the old one.

**Comment by jeremybmerrill** [comment]
Yes, I'm totally down to move to PDF.js -- though I'll let you handle it, since you've done a lot of research into the internals and I wouldn't know where to start.

The implementation effort to increase thumbnails is pretty much nil (changing `560` to `800` is all it is). Displaying users' library PDFs at scaled-up resolutions is I think okay. I'll just switch this to use the 560px if 800px doesn't exist: f3f8a76

**Comment by jeremybmerrill** [comment]
Oops that doesn't actually fix it. :unamused:

**Comment by jeremybmerrill** [comment]
here we go 540aac3

**Comment by jeremybmerrill** [comment]
>  3. You've increased the size of the thumbnails, but their resolution stays low, so it gives a very blurred feeling. Want to consider making them bigger? Or, want to vary it according to number of pages? (more pages, low res)
> 4. It would be great to be able to shrink the page thumbnails to "fit page height" level, so if the table's easily markable, I can scroll to next pages quicker, as before. The bigger view (fit page width) is also useful to have for more precise marking. So I'd suggest keeping a dropdown menu on the top bar, right side with "fit page width", "fit whole page" view options.

User feedback from https://github.com/tabulapdf/tabula/issues/303 says this fix isn't good enough yet.
